Dockerfile

```diff
@@ -7,25 +7,23 @@ FROM ubuntu:18.04 AS raspberrypi
 RUN apt-get update \
     && apt-get upgrade -y \
     && apt-get install -y \
-    # Required by most scripts (when running on the rpi)
-    sudo \
-    wget \
-    # Required by create-vm.ps1 (ssh-keygen, ssh-keyscan)
-    openssh-client \
-    # Installed by mount-vm-share.sh (preinstalling speeds up testing of the script)
-    sshfs \
-    # Cleanup
-    && rm -rf /var/lib/apt/lists/*
-
-# Mimic RaspberryPi: Create a user called "pi" and default password "raspberry" that's in the groups pi and sudo
-RUN useradd --create-home pi --groups sudo --gid root \
-    # && echo 'pi:raspberry' | chpasswd \
-    && echo "pi ALL=(ALL) NOPASSWD:ALL" > /etc/sudoers.d/pi \
-    && mkdir -p /home/pi \
-    && chmod -R g+w /home/pi \
-    && sudo chmod g+w -R /home/pi \
-    && groupadd pi \
-    && usermod -aG pi pi
+    # Required to run image as non-root user
+    sudo
+
+# skel files are used as the basis for new users and include .bashrc etc:
+# https://www.raspberrypi.org/documentation/linux/usage/users.md
+# These files were copied from a RetroPie 4.5.1 on a RaspberryPi 3 and
+# are included to make the image more realistic. One difference from
+# base docker image is that .bashrc has colors enabled.
+COPY docker/rpi/etc/skel /etc/skel
```

Comment:
When building on Windows this will change the line endings to `\r\n` which results in this error when opening a shell:
```
bash: $'\r': command not found
bash: /home/pi/.bashrc: line 6: syntax error near unexpected token `$'in\r''
'ash: /home/pi/.bashrc: line 6: `case $- in
```

It's a known issue since November 2017 so it's likely not going to get fixed now:
https://github.com/docker/for-win/issues/1340

The images built by Docker Hub work fine, so I'll revisit this when adding support for building on multiple architectures.